src/components/page-list/page-list.css

```diff
@@ -1,32 +1,69 @@
-.page-list {
-  table-layout: fixed;
-}
-
-.page-list > tbody > tr {
-  cursor: pointer;
-}
-
-.page-list > tbody > tr:hover {
-  background-color: #eee;
-}
-
-.page-list > tbody > tr > td {
-  overflow-wrap: break-word;
-  word-wrap: break-word;
-}
-
-.page-list > tbody > tr > th[data-name="capture-date"] {
-  width: 12%;
-}
-
-.page-list > tbody > tr > th[data-name="site"] {
-  width: 15%;
-}
-
-.page-list > tbody > tr > th[data-name="page-name"] {
-  width: 33%;
-}
-
-.page-list > tbody > tr > th[data-name="url"] {
-  width: 40%;
-}
+.container {
+  padding: 0 15px;
+}
```

Comment:
Seems like we have this same style that got extracted out of Bootstrap in different ways in a couple different places in this PR. Should it be abstracted to a common module or variable or something? (Or if unclear, maybe mark with a `TODO:` or `FIXME:` comment line for revision after this PR.)

Reply:
Yeah, I didn't want to copy the whole bootstrap grid system, and it seems out of the scope of this PR to impose another system, so I made specific styles for each component on a case-by-case basis.
padding: `0 15px` was showing up a bunch though, so I've now pulled that out into a variable and left a `TODO` note

Reply:
Perfect, that makes a lot of sense. 💯